src/parse.c

```diff
@@ -5245,13 +5245,51 @@ static class_t p_entity_class(void)
    }
 }
 
+static ident_t p_entity_designator()
+{
+   // entity_tag [ signature ]
+   ident_t id = p_identifier();
+
+   if (peek() == tLSQUARE) {
+      type_t constraints = p_signature();
+   }
+   return id;
+}
+
+static ident_list_t *p_entity_name_list(void)
+{
+   // entity_designator { , entity_designator } | others | all
+   ident_list_t *result = NULL;
+
+   switch (peek()) {
+   // LRM-2019 7.2 - Attribute Specification
+   case tOTHERS:
+      // Attribute applies to the names entities of the class declared in the
+      // immediately enclosing declarative part, provided that each such entity
+      // is not explicitly named in the entity name list of the previous
+      // attribute specification for the given attribute. Error if this isn't
+      // the last attribute for the given entity class.
```

Comment:
I don't think there's any need to have all this text here, and these rules wouldn't be checked by the parser at any rate.